src/main/resources/application-dev.yaml

```diff
@@ -33,3 +33,18 @@
             authorization-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/oauth2/v2.0/authorize
             token-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/oauth2/v2.0/token
             jwk-set-uri: https://hmctsdartsb2csbox.b2clogin.com/hmctsdartsb2csbox.onmicrosoft.com/B2C_1_darts_externaluser_signin/discovery/v2.0/keys
+
+ logging:
+   level:
+     uk.gov.hmcts.darts: DEBUG
+     org.springframework.web.servlet.mvc.method.annotation: DEBUG
+     org.springframework.web: DEBUG
+
+ spring.datasource.url: "jdbc:postgresql://localhost:5432/dartsdb"
```

Comment:
These properties probably need to be yamlified - or do we drop them for time being if not being used yet?
Dependent on DMP-45 build / infra stuff

Reply:
Hadn't realised some of these were set elsewhere, so i have removed them all for now, as Scott's work should supersede it anyway. 
not sure about yamlifing the spring.logging elements, as i find that they are much easier to reach when in the dot format. Happy to yamlify them for consistency. 
@jackmaloney any preference?

Reply:
I prefer the properties in dot notation - but honestly didn't think we could have this style in yaml file. Interesting mix. :)

Reply:
yeah, works fine. ok, if you are happy i will resolve comment